src/amo/pages/Home/index.js

```diff
@@ -32,8 +32,8 @@ import Icon from 'ui/components/Icon';
 import './styles.scss';
 
 export const FEATURED_COLLECTIONS = [
-  { slug: 'news-boosters', username: 'mozilla' },
-  { slug: 'weather-reports', username: 'mozilla' },
+  { slug: 'news-boosters', userId: config.get('mozillaUserId') },
```

Comment:
Do you think it would be worth defining a `const` above this for `mozillaUserId = config.get('mozillaUserId')`? There may be times when even more than 2 collections may be displayed on the home page, and this could remove duplication.

Reply:
:+1: